### PR TITLE
Update group-heating-and-cooling-coils.tex

### LINKED ISSUE #9370: Add documentation to Coil:Heating:DX:SingleSpeed field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation
The `Minimum Outdoor Dry-Bulb Temperature for Compressor Operation` field in the `Coil:Heating:DX:SingleSpeed` object is missing documentation. See the [I/O reference section here](https://github.com/NREL/EnergyPlus/blob/develop/doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex#L4063-L4064).

This field appears blank in the published E+ v9.6 I/O reference .pdf.

Somehow it [is documented in the Big Ladder Software online docs](https://bigladdersoftware.com/epx/docs/9-6/input-output-reference/group-heating-and-cooling-coils.html#field-minimum-outdoor-dry-bulb-temperature-for-compressor-operation):
 _"This numeric field defines the minimum outdoor air dry-bulb temperature where the cooling coil compressor turns off. If this input field is left blank, the default value is -25 °C."_ 
This description is incorrect as the E+ v9.6 .idd lists the default temperature as -8°C, not -25 °C.

### PR DESCRIPTION
**Pull request by mdahlhausen**
add documentation to Minimum Outdoor Dry-Bulb Temperature for Compressor Operation field of the Coil:Heating:DX:SingleSpeed object

Pull request overview
---------------------
 - Fixes #9370 
 - Add documentation to Coil:Heating:DX:SingleSpeed field Minimum Outdoor Dry-Bulb Temperature for Compressor Operation

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex [hunk 1]
```diff
@@ -4062,6 +4062,7 @@ \subsubsection{Inputs}\label{inputs-19-001}
 
 \paragraph{Field: Minimum Outdoor Dry-Bulb Temperature for Compressor Operation}\label{field-minimum-outdoor-dry-bulb-temperature-for-compressor-operation}
 
+This numeric field defines the minimum outdoor air dry-bulb temperature where the heating coil compressor turns off. If this input field is left blank, the default value is -8 °C.
```
Comment (Myoldmopar) [inline doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:4065]: I haven't verified the -8C, but this change in general seems totally fine.

Comment (Nigusse) [reply doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:4065]: @Myoldmopar this field all DX heating coils have -8.0C as the minimum value while all DX cooling coils have -25.0C based on the latest idd.

Comment (Myoldmopar) [reply doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex:4065]: Thanks for confirming @Nigusse

**Comment by Myoldmopar** [comment]
Thanks for this documentation fix @mdahlhausen.  Merging.

**Pull request merged**